packages/web/src/components/share-modal/ShareModal.tsx

```diff
@@ -48,6 +52,19 @@ export const ShareModal = () => {
   const isOwner =
     content?.type === 'track' && account?.user_id === content.artist.user_id
 
+  const handleShareToDirectMessage = useCallback(async () => {
+    if (!content) return
+    dispatch(setVisibility({ modal: 'CreateChat', visible: true }))
+    dispatch(
+      setCreateChatModalState({
+        // Just care about the link
+        presetMessage: (await getTwitterShareText(content, false)).link,
```

Comment:
Will this include "#audius" in the string? is that weird?

Reply:
nope, this is just the link part of it!